learning_journal/models/mymodel.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Text,
+    UnicodeText,
+    Date
+)
+
+from .meta import Base
+import datetime
+
+
+def _now():
+    return datetime.datetime.now()
+
+
+class Journal(Base):
+    __tablename__ = 'journals'
+    id = Column(Integer, primary_key=True)
+    title = Column(Text)
```

Comment:
I think you probably want to use `Unicode` here, rather than `Text`.  The difference lies in what type of data you are expected to use in Python.  the `Unicode` type refuses to accept bytes.  You really want that type of behavior, I think.